src/mongocrypt.h

```diff
@@ -712,6 +712,7 @@ bool mongocrypt_ctx_setopt_algorithm(mongocrypt_ctx_t *ctx, const char *algorith
 // DEPRECATED: support "RangePreview" has been removed in favor of "range".
 #define MONGOCRYPT_ALGORITHM_RANGEPREVIEW_DEPRECATED_STR "RangePreview"
 #define MONGOCRYPT_ALGORITHM_RANGE_STR "Range"
+#define MONGOCRYPT_ALGORITHM_TEXTPREVIEW_STR "textPreview"
```

Comment:
```suggestion
/// NOTE: "textPreview" is experimental only and may be removed in a future non-major release.
#define MONGOCRYPT_ALGORITHM_TEXTPREVIEW_STR "textPreview"
```

Suggested warning follows the warning above the `MONGOCRYPT_QUERY_TYPE_*` constants.